examples/seismic/acoustic/acoustic_example.py

```diff
@@ -70,8 +57,11 @@ def run(shape=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=1000.0,
                             space_order=space_order, kernel=kernel,
                             constant=constant, **kwargs)
 
-    initial_vp = smooth10(solver.model.m.data, solver.model.shape_domain)
-    dm = np.float32(initial_vp**2 - solver.model.m.data)
+    # Smooth velocity
+    initial_vp = Function(name='ivp', grid=solver.model.grid, space_order=space_order)
```

Comment:
would still name it `v0`, `ivp` seems weird (nitpicking)

Reply:
it was called ``initial_vp``, happy to change it to v0 !

Reply:
done